### DISCUSSION TITLE
捣鼓直连时发现件荒唐事，CFIEC 的 DNS64 居然能翻墙

### DISCUSSION THREAD

**Discussion by SeaHOH** [SeaHOH/GotoX#102] [一般]
http://www.cfiec.net/dns/s/?978.html

只对没有 IPv6 的域名有效，这个 NAT64 网关也比较特别，IP 没有一一对应。
简单试了下，速度还好。

搜了下相关内容，发现这个情况起码已经持续一年以上。
没搞明白他们的逻辑，难道会随机进行中间人攻击？

**Comment by ghost** [comment]
具体是可以访问哪些网站呢？

**Reply by SeaHOH** [reply]
脸书之类的都行，但是好像现在已经不行了。
